### ISSUE TITLE
R8000 bricked.

### ISSUE THREAD

**Issue by unknownx2** [jclehner/nmrpflash#99]
I have a R8000 that i bricked. I am trying to use this software on a linux. I got it to work. I am getting error message no response after 60 seconds. Where should i be plugging the ethernet cable, on the WAN port which is yellow or LAN1, LAN2 LAN3? Ive tried WAN and LAN1. Any suggestions please?

**Comment by toastom** [comment]
I have an R7000P but it should be the same process. Plug it into one of the LAN ports, probably the first.

**Comment by jclehner** [comment]
Yes, it's most likely LAN1. Also, make sure to turn the router **off** before running `nmrpflash`.

**Comment by jclehner** [comment]
Try the latest version, and also check out #89.

**Comment by unknownx2** [comment]
Thanks i will download the latest version. My attempt was unsuccessful, but refuse to give up

**Comment by jclehner** [comment]
If you still get the "no response after 60 seconds" error, try this attached version:

[nmrpflash-0.9.17-4-g784536c-linux.zip](https://github.com/jclehner/nmrpflash/files/8927444/nmrpflash-0.9.17-4-g784536c-linux.zip)

**Comment by jclehner** [comment]
Closed due to inactivity.
